pyvista/utilities/errors.py

```diff
@@ -135,9 +207,17 @@ def __init__(self, additional=None, ncol=3, text_width=80, sort=False):
         optional = ['matplotlib', 'PyQt5', 'IPython', 'colorcet',
                     'cmocean', 'panel']
 
+        # Information about the GPU - bare except incase there is a rendering
+        # bug that the user is trying to report.
+        try:
+            extra_meta = GPUInfo().get_info()
+        except:
+            extra_meta = ("GPU Details", "error")
+
```

Comment:
We may want to have a boolean flag to skip this in case the user is experiencing major issues with rendering

Reply:
done